Mergin/create_project_wizard.py

```diff
@@ -0,0 +1,249 @@
+import os
+
+from qgis.PyQt import uic
+from qgis.PyQt.QtCore import QSettings, Qt, QSize
+from qgis.PyQt.QtWidgets import QApplication, QWizard, QFileDialog
+
+from qgis.core import QgsProject
+from qgis.utils import iface
+
+from .utils import create_basic_qgis_project, find_qgis_files
+
+base_dir = os.path.dirname(__file__)
+uicls_init_page, basecls_init_page = uic.loadUiType(os.path.join(base_dir, "ui", "ui_new_proj_init_page.ui"))
+uicls_local_path_page, basecls_local_path_page = uic.loadUiType(os.path.join(base_dir, "ui", "ui_get_path_page.ui"))
+uicls_proj_settings, basecls_proj_settings = uic.loadUiType(os.path.join(base_dir, "ui", "ui_project_settings_page.ui"))
+
+INIT_PAGE = 0
+SAVE_PAGE = 1
+LOCATE_PAGE = 2
+SETTINGS_PAGE = 3
+PACK_PAGE = 4
+
+
+class InitPage(uicls_init_page, basecls_init_page):
+    """Initial wizard page with Mergin project sources to choose."""
+
+    def __init__(self, parent=None):
+        super().__init__(parent)
+        self.setupUi(self)
+        self.parent = parent
+        self.btns_page = {
+            self.basic_proj_btn: SAVE_PAGE,
+            self.cur_proj_no_pack_btn: LOCATE_PAGE,
+            # self.cur_proj_pack_btn: SAVE_PAGE,
+        }
+        for btn in self.btns_page.keys():
+            btn.setAutoExclusive(True)
+            btn.clicked.connect(self.selection_changed)
+        self.hidden_ledit.hide()
+        self.registerField("create_from*", self.hidden_ledit)
+
+    def selection_changed(self):
+        self.hidden_ledit.setText("Selection done!")
+
+    def nextId(self):
+        """Decide about the next page based on checkable buttons."""
+        next_id = INIT_PAGE
+        for btn in self.btns_page.keys():
+            if btn.isChecked():
+                next_id = self.btns_page[btn]
+                break
+        # make sure current project is saved, if not, open save page instead of locate
+        proj_path = QgsProject.instance().absoluteFilePath()
+        if next_id == LOCATE_PAGE and not proj_path:
+            next_id = SAVE_PAGE
+        return next_id
+
+
+class ChoosePathPage(uicls_local_path_page, basecls_local_path_page):
+    """Page for getting local path for saving new Mergin project."""
+
+    def __init__(self, question=None, existing=None, file_filter=None, parent=None):
+        super().__init__(parent)
+        self.setupUi(self)
+        self.parent = parent
+        self.file_filter = "QGIS projects (*.qgz *.qgs)" if file_filter is None else file_filter
```

Comment:
QGIS open project have also *.QGZ *.QGS listed in filters. Could you add them as well?